fluss-lake/fluss-lake-paimon/src/test/java/org/apache/fluss/lake/paimon/tiering/PaimonTieringITCase.java

```diff
@@ -489,6 +489,93 @@ private String getPartitionOffsetStr(Map<Long, String> partitionNameByIds) {
         return "[" + String.join(",", partitionOffsetStrs) + "]";
     }
 
+    private static Stream<Arguments> tieringNestedRowWriteArgs() {
+        return Stream.of(Arguments.of(true), Arguments.of(false));
+    }
+
+    @ParameterizedTest
+    @MethodSource("tieringNestedRowWriteArgs")
```

Comment:
Replace by `@ValueSource(booleans = {true, false})` and remove `tieringNestedRowWriteArgs` method.